tools/chaincode-integration/src/step-definitions/chaincode/chaincode.ts

```diff
@@ -307,7 +307,7 @@ export class Chaincode {
 
         const attempts = 10;
 
-        await sleep(2000);
+        await sleep(20000);
```

Comment:
Is there a reason to up this from checking every 2 seconds? If more time is needed I'd rather update the attempts from 10